app/handlers/trajectoryAnalysisStatus.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+import tornado.escape
+import tornado.web
+from utils.statusHelper import sharedStatusHelper
+
+class TrajectoryAnalysisStatusHandler(tornado.web.RequestHandler):
```

Comment:
How is TrajectoryAnalysisStatusHandler and SafetyAnalysisStatusHandler different?  As I see it, the user sends a 'videoId' and the sharedStatusHelper tells them where it is in the process. I can see one handler doing all of that, and not needing to specify two different endpoints.